src/containers/sound-editor.jsx

```diff
@@ -165,7 +165,7 @@ class SoundEditor extends React.Component {
             )
             .catch(e => {
                 // Encoding failed, or the sound was too large to save so edit is rejected
-                log.error(`Encountered error while trying to encode sound update: ${e}`);
+                log.error(`Encountered error while trying to encode sound update: ${e.message}`);
```

Comment:
Wow, did lint catch this? If so, that's great!

Reply:
Alas no, that was me tracing where the rejections were going.